### DISCUSSION TITLE
Add radio sizes, like already present in checkbox

### DISCUSSION THREAD

**Discussion by stefanobartoletti** [saadeghi/daisyui#3198] [Ideas / feature requests]
Checkbox component has different classes for different sizes, as documented here https://daisyui.com/components/checkbox/#sizes

I think it would be nice to have the same thing even for Radio inputs, so that it is possible to build a consistent form

**Comment by saadeghi** [comment]
Radio has same sizes:   
<img width="566" alt="Screenshot 2024-09-05 at 10 07 09 PM" src="https://github.com/user-attachments/assets/b462f522-4063-499d-aee1-7b964a80f90e">

The class names are listed in docs: https://daisyui.com/components/radio/  But the examples are missing. I will add examples. Thanks!

**Reply by stefanobartoletti** [reply]
Thanks, I didn't notice the modifiers were already there
